x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -272,7 +272,7 @@ export const EndpointList = () => {
       },
       {
         field: 'host_status',
-        width: '9%',
+        width: '14%',
```

Comment:
thank your for adjusting this!

Reply:
it was driving me nuts when there were more than one action pending. we might need to adjust further :)

Reply:
Checked it out locally on my small mac screen.  The common statuses look good, however the multiple pending status case is still cut off.

![image](https://user-images.githubusercontent.com/56395104/122945339-848df780-d346-11eb-88b2-297861b2de16.png)

I think we could still take a bit more width from hostname and/or ips if we wanted to give it more room.  The tooltip helps with the multiple pending status, though.

![image](https://user-images.githubusercontent.com/56395104/122945675-bacb7700-d346-11eb-91bd-23ecd2680fe3.png)